pkg/apis/core/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
how was this file generated?

Reply:
`make genereate-go` I believe

https://github.com/tinkerbell/tink/blob/main/kube.mk#L25-L26

Reply:
Ah I see. I also just noticed that it says so right in the file https://github.com/tinkerbell/tink/pull/563/files#diff-228d0bde90a51d1ee8bc16fa0c9003354a52185a5898d6e6f47fcf1ba426221bR20 :smile: . I've opened up https://github.com/tinkerbell/tink/pull/564 to rework kube.mk to be simpler imo, can you take a look please?